runtime/oti/j9nonbuilder.h

```diff
@@ -337,6 +337,11 @@
 #define J9_VISIBILITY_NEST_MEMBER_NOT_CLAIMED_ERROR -5
 #endif /* defined(J9VM_OPT_VALHALLA_NESTMATES) */
 
+typedef enum{
+	DataBreakPoint
```

Comment:
The enum vaulues could probably use some decoration.

Reply:
change the enum name to be `J9DataBreakpointFlushCompQueueReason`

Reply:
Personally, I'd name it with a common prefix, and the reason at the end.

Reply:
Changed according to suggestion.